ballerina/readable_data_channel.bal

```diff
@@ -83,7 +83,7 @@ public class ReadableDataChannel {
     # boolean|io:Error result = dataChannel.readBool();
     # ```
     #
-    # + return - boolean value which is read or else `io:Error` if any error occurred
+    # + return - Boolean value which is read or else `io:Error` if any error occurred
```

Comment:
```suggestion
    # + return - Boolean value, which is read or else `io:Error` if any error occurred
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/477/commits/b940cf5754398095dd38803252c05359a666001c